app/Http/Controllers/Auth/LoginController.php

```diff
@@ -127,4 +129,27 @@ public static function staticShowLoginForm()
                 'js_version' => self::getJsVersion()
             ]);
     }
+
+    public static function saml2Login() : \Illuminate\Http\Response|\Illuminate\Http\RedirectResponse
```

Comment:
The return type can be cleaned up by importing the relevant class.  `\Illuminate\Http\Response` is already being used, and `\Illuminate\Http\RedirectResponse` can be added.  The end result would be a return type of `Response|RedirectResponse` which is a little bit easier to read.